### DISCUSSION TITLE
Acceleration settings not being applied

### DISCUSSION THREAD

**Discussion by DarthMaledous** [OrcaSlicer/OrcaSlicer#260] [Q&A]
I ran a couple of test prints with the latest version of Soft Fever and ran into a weird issue. No matter what I do, it won't print above 500mm/s acceleration. I took a look at the gcode in the file it sliced and I can see where it's hard setting the acceleration to 500mm/s. Any ideas?

**Comment by hliebscher** [comment]
can you post the g-code?

**Comment by SoftFever** [comment]
Pls check your machine limits in the printer settings.
